target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10mp.dts

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+#include "rtl83xx_d-link_dgs-1210_poe.dtsi"
+
+/ {
+	compatible = "d-link,dgs-1210-10mp", "realtek,rtl838x-soc";
```

Comment:
```suggestion
	compatible = "d-link,dgs-1210-10mp", "realtek,rtl8382-soc", "realtek,rtl838x-soc";
```
allows you at least to have some quirck/overrides if needed be

Reply:
This has been changed.